x-pack/plugins/fleet/server/services/agent_policies/package_policies_to_agent_permissions.ts

```diff
@@ -247,3 +261,26 @@ function apmPermissions(packagePolicyId: string): [string, SecurityRoleDescripto
     },
   ];
 }
+
+function connectorServicePermissions(packagePolicyId: string): [string, SecurityRoleDescriptor] {
+  return [
+    packagePolicyId,
+    {
+      cluster: ['manage_connector'],
+      indices: [
+        {
+          names: ['traces-*', 'logs-*', 'metrics-*'],
```

Comment:
Not familiar with the connector but does it need to write to all of those  logs, metrics, traces datastreams?

Reply:
The connector component doesn't need to write to those indices.

IIRC when I was testing locally, I *think* that I could only access the connector component logs in Fleet UI after adding this to permissions. So, I think it does need to write to `logs-*` at least (could I be wrong here?). I'm not sure about `traces-*` and `logs-*` honestly.

Reply:
I think logs will be send by elastic-agent if  monitoring is enabled for the agent policy, this how it works for other components, so unless there is a specific need I think we probably not need those permissions

Reply:
@nchaulet Actually after removing this bit we are no longer able to receive component-level logs, even with system logs + metrics collection enabled, I'm leaning towards adding:
```
{
  names: ['logs-elastic_agent*'],
  privileges: ['auto_configure', 'create_doc']
}
```